### ISSUE TITLE
please help me, Can I trained with the data whose size is 608*608*20.

### ISSUE THREAD

**Issue by electronicYH** [AlexeyAB/darknet#5489]
Thank you for your work.
I have a job to train a abnormal dataset. 
Data in the datatset has the size: W:608,H:608,C:20, the size is different from normal image format.
please help me, how can I train the datset with YOLOv4.
Thank you very much.

**Comment by lsd1994** [comment]
What's the meaning of those channels? Or you can separate them into 20 images by 608*608*1

**Comment by electronicYH** [comment]
yes, I mean the channels number is 20. And it is can't be separated into 20 images by 608*608*1.
So, how can I train YOLOV4 with the data?
thank you!
